reactive/context.ts

```diff
@@ -228,13 +506,78 @@ export class Context<T extends record = record> extends EventTarget {
     if ((type === "set") || (type === "delete") || (type === "call")) {
       this.dispatchEvent(new Context.Event("change", { detail }))
     }
+
+    this.dispatchEvent(new Context.Event("all", { detail }))
```

Comment:
I added it because there is a chance that a person might want to listen to "call" events as well, and I thought needing to create 3 event handlers would be kinda annoying but all in all its not very neccessary, I'll remove it.